testsuite/domain/src/test/resources/host-configs/host-slave-auto-start.xml

```diff
@@ -78,7 +78,7 @@
 
     <domain-controller>
         <!-- Remote domain controller configuration with a host and port -->
-        <remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm">
+        <remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm" ignore-unused-configuration="false">
```

Comment:
That's odd, this isn't set in mine:

<remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm">

I'm just about ready to push again, I'll verify its not there after.